Source/Generator/README.md

```diff
@@ -3,53 +3,40 @@
 
 #### Make Changes to the Measurement Logic
 
-Measurement Logic for the measurements are coded under `Measurement Logic.vi` under the respective libraries. The measurement logic can be modified in it. After modifying the measurement logic, run the measurement by following the `Run the Measurement Service` steps from above.
+Measurement Logic for the measurement service is located in `Measurement Logic.vi` under the measurement library. After modifying the measurement logic, run the measurement by following the `Run the Measurement Service` steps from above.
 
 #### How to add new UI elements
 
-User Interface is created in `Measurement UI.vi` under the `<LibraryName>.lvlib\`. The control and indicator names in the User Interface should match the `Measurement Configuration` and `Measurement Results`. If the datatype and name matches, then the data from the UI will sent to the logic before execution and the results will be published to UI after measurement is run.
-
-## Notes
-
-### How GetMetaData and DoMeasure are triggered in the Measurement Service
-
-This is abstracted in the library, and a developer need not know how the frameworks request and recieve the response from the measurement client, and when in measurement client that Get MetaData and Measure functions are called. To understand the Measurement Service please find the details below,
-
-1. To understand the LabVIEW Service and how a service function is called on client call, please refer: https://github.com/ni/grpc-labview/blob/master/docs/BasicsTutorial.md
-
-Measurement Service and the function calls are similar to how the general LabVIEW service works as mentioned in the link above. You can find the `MeasurementService` and `ServiceManagementService` under the `<libraryName>.lvlib\Service\`. 
-
-`MeasurementService` is responsible for responding to the MetaData request and calling the `Measurement Logic.vi` when the DoMeasurement request is sent from the frameworks. These implementation can be found in: `Start Sync.vi`
-
-`ServiceManagementService` is responsible for managing the service like triggering stop event when the framework request to stop the service. These implementation can be found in: `Start Sync.vi`
+The User Interface for a measurement service is defined by the front panel of `Measurement UI.vi` under the `<LibraryName>.lvlib\`. The control and indicator names on the front panel should match the `Measurement Configuration` and `Measurement Results`. If the datatype and name matches, data from the front panel controls will be sent to the logic before execution and the results will be published to the front panel indicators after the measurement is run.
```

Comment:
A few comments here:
- Rather than control and indicator names, I would say control and indicator labels. That way customers won't confuse labels and captions.
- We are inconsistent in how we use <LibraryName>.lvlib. Above this, we use <MeasurementName>.lvlib

Reply:
Updated to be consistent on MeasurementName.lvlib.